app/src/main/java/org/mozilla/fenix/settings/creditcards/controller/CreditCardEditorController.kt

```diff
@@ -31,9 +32,9 @@ interface CreditCardEditorController {
     fun handleDeleteCreditCard(guid: String)
 
     /**
-     * @see [CreditCardEditorInteractor.onSaveCreditCard]
+     * @see [CreditCardEditorInteractor.onCreateCreditCard]
      */
-    fun handleSaveCreditCard(creditCardFields: UpdatableCreditCardFields)
+    fun handleCreateCreditCard(creditCardFields: NewCreditCardFields)
```

Comment:
I would prefer to keep this as `handleSaveCreditCard` to keep with the wording in the UI. It just makes it a bit clearer when someone sees the "Save" button and correlate it with `handleSaveCreditCard`

```suggestion
    fun handleSaveCreditCard(creditCardFields: NewCreditCardFields)
```

Reply:
Sure!